### PR TITLE
cv ulpoad flow - fixed bug, replaced == with ===

### PR DESCRIPTION
**Pull request by wmosienski**

### CODE REVIEW

File: client/src/HomePage/CvUploadFlow.jsx [hunk 1]
```diff
@@ -96,17 +96,17 @@ export const CvUploadFlow = (props) => {
     
     const addSkill = (name, value) => {
         if (name !== "" && skillsDictionary.filter((el) => !skills.map((el2) => el2.name).includes(el)).includes(name)) {
-            setSkills(skills => [...skills, {name: name, value, value}]);
+            setSkills(skills => [...skills, {name: name, value: value}]);
```
Comment (solleOlle) [inline client/src/HomePage/CvUploadFlow.jsx:99]: why don't you do it: `[...skills, { name, value }])`?

Comment (wmosienski) [reply client/src/HomePage/CvUploadFlow.jsx:99]: oki, im changing it :)

Comment (solleOlle) [reply client/src/HomePage/CvUploadFlow.jsx:99]: thanks! :)

File: client/src/HomePage/CvUploadFlow.jsx [hunk 2]
```diff
@@ -96,17 +96,17 @@ export const CvUploadFlow = (props) => {
...
         }
         
     }
 
     const updateValue = (name, value) => {
-        setSkills(skills => [...skills.map((el, idx) => el.name == name ? {name: name, value: value} : el)]);
+        setSkills(skills => [...skills.map((el, idx) => el.name === name ? {name: name, value: value} : el)]);
```
Comment (solleOlle) [inline client/src/HomePage/CvUploadFlow.jsx:105]: the same here

Comment (wmosienski) [reply client/src/HomePage/CvUploadFlow.jsx:105]: yep, all will be in a next pull request, because Marcin merged this one :c

**Pull request merged**